ote/src/cljs/ote/views/admin/monitor.cljs

```diff
@@ -81,55 +82,68 @@
                                                      (mapv :sub-type (:companies-by-service-type monitor-data)))
                                        :datasets [{:data (mapv :count (:companies-by-service-type monitor-data))
                                                    :backgroundColor ["rgb(0,170,187)"
-                                                                     "rgb(102,214,184)"
                                                                      "rgb(102,204,102)"
+                                                                     "rgb(153,204,0)"
                                                                      "rgb(221,204,0)"
                                                                      "rgb(255,136,0)"
                                                                      "rgb(255,102,153)"
-                                                                     "rgb(235,102,204)"]
+                                                                     "rgb(153,0,221)"]
```

Comment:
Shouldn't all colors be defined in style file, colors.cljs

Reply:
It will make code much readable. I'll make the change.